pandas/tests/plotting/test_series.py

```diff
@@ -729,6 +729,22 @@ def test_dup_datetime_index_plot(self):
         s = Series(values, index=index)
         _check_plot_works(s.plot)
 
+    def test_errorbar_asymmetrical(self):
+        # GH9536
+        s = Series(np.arange(10), name="x")
+        err = np.random.rand(2, 10)
+
+        ax = s.plot(yerr=err, xerr=err)
+
+        yerr_0_0 = ax.collections[1].get_paths()[0].vertices[:, 1]
+        expected_0_0 = err[:, 0] * np.array([-1, 1])
+        tm.assert_almost_equal(yerr_0_0, expected_0_0)
+
+        with pytest.raises(ValueError):
```

Comment:
Could you please check the error message too?

Reply:
Added in https://github.com/pandas-dev/pandas/pull/34514/commits/99b51fc91279a486b9fe56eab6df6f93f40d292f and https://github.com/pandas-dev/pandas/pull/34514/commits/0ea50d35ec302c0015c9cabedb77b8b91b1a41df